docs/book/how-does-opa-work.md

```diff
@@ -395,7 +393,7 @@ violations[server] {
     # ...and any of the server’s protocols is HTTP
     server.protocols[_] = "http"
     # ...and the server is public.
-    public_servers[server] = true
```

Comment:
Nice catch. This example goes back to the beginning of the project. Originally, we defined sets such that `p[x] = true`. We found this confused people so we eventually changed sets such that `p[x] = x`.